x-pack/platform/plugins/private/canvas/public/components/color_picker/color_picker.tsx

```diff
@@ -28,18 +28,17 @@ export const ColorPicker: FC<Props> = ({
   onRemoveColor,
   value = '',
 }) => {
-  const tc = tinycolor(value);
-  const isValidColor = tc.isValid();
+  const isValidColor = chroma.valid(value);
 
   colors = colors.filter((color) => {
-    return tinycolor(color).isValid();
+    return chroma.valid(color);
   });
 
   let canRemove = false;
   let canAdd = false;
 
   if (isValidColor) {
-    const match = colors.filter((color) => tinycolor.equals(value, color));
+    const match = colors.filter((color) => chroma(value).name() === chroma(color).name());
```

Comment:
This doesn't work if the color has an alpha value and has a color keyword

```
chroma("rgb(0 128 128 / 0.5)").name();
'teal'
chroma("rgb(0 128 128)").name()
'teal'

chroma("rgb(0 128 128 / 0.5)").hex();
'#00808080'
chroma("rgb(0 128 128)").hex()
'#008080'
```

better compare the hex values

```suggestion
    const match = colors.filter((color) => chroma(value).hex() === chroma(color).hex());
```